doc/conf.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#
+# ToscaWidgets 2 documentation build configuration file, created by
```

Comment:
Lol.  ToscaWidgets.  I will never escape you!

Reply:
Haha. Well, at least it's not the original TurboGears widgets like we have in Beaker :-)